connexion/app.py

```diff
@@ -85,8 +86,13 @@ def common_error_handler(exception):
         """
         :type exception: Exception
         """
+        # If we have an instance of problem exception we can use the title to render a more complete title.
+        if isinstance(exception, ProblemException) and isinstance(exception, werkzeug.exceptions.HTTPException):
```

Comment:
Kinda. `ProblemException` actually needs to be combined with a direct subclass of `werkzeug.exceptions.HTTPException` (but your right my code doesn't achieve that). As several functions in the `werkzeug.exceptions.HTTPException` will fail without a status code. Hence why my 3 OAuth execption classes combine a `werkzeug.exceptions.HTTPException_SUBCLASS` with a `ProblemException`